org.kde.konquest.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.konquest",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "konquest",
+    "rename-icon": "konquest",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-config/kdeglobals:ro"
```

Comment:
```suggestion

```

Reply:
Why?

Reply:
https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/5303ce4e8f74683f280d3d87bfe33d4776c6c9a8/org.kde.Sdk.json.in#L76